.changeset/famous-walls-visit.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-search': minor
+'@backstage/plugin-search-react': minor
+---
+
+Components `<DefaultResultListItem>`, `<SearchBar>` (including `<SearchBarBase>`), `<SearchFilter>` (including `.Checkbox`, `.Select`, and `.Autocomplete` static prop components), `<SearchResult>`, `<SearchResultPager>`, and `<SearchTracker>` are now exported from `@backstage/plugin-search-react`. They are now deprecated in `@backstage/plugin-search` and will be removed in a future release.
```

Comment:
Can probably remove `<SearchTracker>` from here too.

Reply:
I think everything is exported from `src/components` so it means `TrackSearch` too. I guess it should just be renamed from `SearchTracker` to `TrackSearch`

Reply:
It doesn't look to have ever been exported from `@backstage/plugin-search` (I don't see a corresponding delete line in its `api-report.md`, at least), so I'm wondering if the export here was by accident.

Reply:
It is used in `<SearchBarBase>`, but I guess I could import and re-export it instead?

Reply:
Alright, if thats the case then we can delete it! Thanks!